rental.rb

```diff
@@ -0,0 +1,13 @@
+require './book'
+require './person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(book, date, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- Hey, while you've added the constructor, we need to also ensure the rental itself is added to the @books and @person to achieve the belongs to the association as the rental is the intermediate between a person and a book. let's add the reming part of the code below to achieve the result.

```
 person.rentals << self
 book.rentals << self
```